frontend/src/views/StartView.vue

```diff
@@ -0,0 +1,217 @@
+      </p>
+      <Button
+        class="btn-apply"
+        @click="enquiryOrIntake = 'enquiry'"
+      >
+        Start enquiry
+      </Button>
+    </div>
+  </div>
+
+  <div
+    v-else-if="enquiryOrIntake"
+    class="container text-left pl-6 pr-6"
+  >
+    <h4 class="h4-title">Notice for collection, use and disclosure of personal information</h4>
+    <div class="p-disclaimer">
+      <p>
+        This information is being collected under the legal authority of section 26 (c) and 27 (1)(a)(i) of the Freedom
+        of Information and Protection of Privacy Act (the Act) and is being used for the purpose of creating a client
+        relationship between you or your organization and Government of British Columbia. It may also be shared when
+        strictly necessary with partner agencies that are also subject to the provisions of the Act. Personal
+        information may be used by the Permitting Solutions Office for survey purposes. If you have any questions
+        regarding the use of this personal information, please contact Housing Authorizations at
+        <a href="mailto:Housing.Authorizations@gov.bc.ca">Housing.Authorizations@gov.bc.ca.</a>
+      </p>
+    </div>
+
+    <h4 class="h4-title mt-2">Before you begin</h4>
+    <div class="p-disclaimer">
+      <p class="p-red">
+        Please connect with your local government for requirements on local zoning, bylaws, and permitting rules. We
+        advise that you engage with First Nations on land and resource decisions as early as possible, if applicable. Go
+        to
+        <a
+          href="https://permitconnectbc.gov.bc.ca/#engaging"
+          target="”_blank”"
+        >
+          Permit Connect BC
+        </a>
+        for more information.
+        <span class="span-underline">
+          Please be aware that the Navigators Service cannot overrule the timelines and decisions of other line of
+          businesses.
+        </span>
+      </p>
+    </div>
+    <div class="flex flex-wrap gap-3 justify-content-between mt-4">
+      <div>
+        <RadioButton
+          v-model="acceptDisclaimer"
+          value="accepted"
+          @click="flipAcceptDisclaimer"
+        />
+        <label
+          for="ingredient1"
+          class="ml-2"
+        >
+          <span
+            class="span-agree"
+            @click="flipAcceptDisclaimer"
+          >
+            <b>Yes, I understand</b>
+          </span>
+          <span class="pl-1 span-red">*</span>
+        </label>
+        <div />
+      </div>
+      <Button
+        label="Start application"
+        outlined
+        class="ml-2"
+        :disabled="!acceptDisclaimer"
+        @click="handleStartApplication"
+      />
+    </div>
+  </div>
+</template>
+
+<style scoped lang="scss">
+.p-breadcrumb {
+  border: none;
+  text-decoration: none;
+}
+
+.no-underline {
+  text-decoration: none;
+}
+
+.p-disclaimer {
```

Comment:
Any particular reason we are using `p-`? This is not a PrimeVue class.

Reply:
Initially I was trying to apply that property to a <p>. I just changed it to div-disclaimer, as it is currently assigned to a <div>. does that work?